packages/realm-react/src/index.tsx

```diff
@@ -19,109 +19,28 @@
 import Realm from "realm";
 import React, { createContext } from "react";
 
-import { createRealmProvider } from "./RealmProvider";
+import {
+  DynamicRealmProvider,
+  RealmProviderFromConfiguration,
+  RealmProviderFromRealmFC,
+  createRealmProvider,
+} from "./RealmProvider";
 import { createUseObject } from "./useObject";
 import { createUseQuery } from "./useQuery";
 import { createUseRealm } from "./useRealm";
+import { RealmContext } from "./RealmContext";
 
 export type { UseObjectHook } from "./useObject";
 export type { UseQueryHook, QueryHookOptions, QueryHookClassBasedOptions } from "./useQuery";
 
-type RealmContext = {
```

Comment:
I believe this was documented here for the docs to generate correctly, did you try to run the docs command to compare the different outputs? I did [upstream an issue](https://github.com/TypeStrong/typedoc/issues/2521) which seems to have been fixed and will be released in [an upcoming `typedoc@0.26.0` release](https://github.com/TypeStrong/typedoc/issues/2567).

Reply:
Ah I see, didn't think about type-based doc generation, will check

Reply:
@kraenhansen where was the issue before? 

I don't see a RealmContext in docs at all since this wasn't being exported before:
Was it with the Provider?
https://www.mongodb.com/docs/realm-sdks/react/latest/functions/RealmProvider.html

From what I can see docs seem largely unchanged 🤞